ajax/libs/simpleCookie/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "simplecookie",
+  "version": "1.0.1",
+  "description": "A small javascript utility to make handling cookies simple",
+  "filename": "simpleCookie.min.js",
```

```suggestion
  "filename": "simpleCookie.js",
```
Only `simpleCookie.js` is in the package, no minified file.